chasm/lib/scheduler/config.go

```diff
@@ -0,0 +1,78 @@
+package scheduler
+
+import (
+	"time"
+
+	"go.temporal.io/server/common/backoff"
+	"go.temporal.io/server/common/dynamicconfig"
+)
+
+type (
+	Tweakables struct {
+		DefaultCatchupWindow              time.Duration // Default for catchup window
+		MinCatchupWindow                  time.Duration // Minimum for catchup window
+		MaxBufferSize                     int           // MaxBufferSize limits the number of buffered actions pending execution in total
+		CanceledTerminatedCountAsFailures bool          // Whether cancelled+terminated count for pause-on-failure
+		RecentActionCount                 int           // Number of recent actions taken (workflow execution results) recorded in the ScheduleInfo metadata.
+		MaxActionsPerExecution            int           // Limits the number of actions (startWorkflow, terminate/cancel) taken by ExecuteTask in a single iteration
+		RetentionTime                     time.Duration // How long to keep schedules after they're done
```

Comment:
interesting... it's not namespace retention? How/where is this used? I don't think CHASM components right now has a way to control their retention time after close.

Reply:
Let's continue the discussion in https://github.com/temporalio/temporal/pull/8165#discussion_r2268010797. Resolving this one.